cmd/create/cluster/cmd.go

```diff
@@ -1497,13 +1500,15 @@ func run(cmd *cobra.Command, _ []string) {
 			if isHostedCP {
 				createAccountRolesCommand = createAccountRolesCommand + " " + hostedCPFlag
 			}
-			r.Reporter.Warnf(fmt.Sprintf("No compatible account roles with version '%s' found. "+
-				"You will need to manually set them in the next steps or run '%s' to create them first.",
-				minor, createAccountRolesCommand))
+			r.Reporter.Warnf("No suitable accounts with ROSA CLI-created account roles were found. "+
+				"You can manually set them in the next steps or run '%s' to create them first.", createAccountRolesCommand)
 			interactive.Enable()
 		}
 
 		if roleARN != "" {
+			// Check if role has red-hat-managed tag
+			roles.CheckRedHatManagedTag(roleARN, awsClient, r)
+
```

Comment:
I think, for this function, we should return a bool, and add the logic here. It's a bit vague at what this does, and what is going on until you look into this function itself (I left a couple small comments on the naming + functionality of it)

Reply:
should be fixed here [6a52f12](https://github.com/openshift/rosa/pull/2867/commits/6a52f123604cf00d48743681600fb9af9538cf64)